sickbeard/tv.py

```diff
@@ -1433,6 +1433,17 @@ def refreshSubtitles(self):
         """Look for subtitles files and refresh the subtitles property"""
         self.subtitles = subtitles.subtitlesLanguages(self.location)
 
+    def getWantedLanguages(self):
+        languages = set()
+        for language in frozenset(subtitles.wantedLanguages()).difference(subtitles.subtitlesLanguages(self.location)):
+            languages.add(subtitles.fromietf(language))
+        self.refreshSubtitles()
+        return languages
+
+        if not languages:
```

Comment:
read it again.

```python
return languages
if not languages:
    logger.log.....
```
Lines 1443-1445 need moved to right after `languages = self.getWantedLanguages();`